ParisBNF/abb/BNFabb160.xml

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb160" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Sawāsǝw-treatise on lexicon and grammar</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+            <date>2016-06-07T17:45:12.922+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+               <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno facs="https://archivesetmanuscrits.bnf.fr/ark:/12148/cc1056302">BnF Éthiopien d'Abbadie 160</idno>
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+
+                             <msItem xml:id="ms_i1">
+                               <locus from="1r" to="73"/>
+                               <title type="complete">Sawāsǝw-treatise on lexicon and grammar</title>
+                               <textLang mainLang="gez"/>
+
+                               <msItem xml:id="ms_i1.1">
```

Comment:
I think it would make sense to add everywhere to textLang `@otherLangs="am"`

Reply:
ok